pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/VenueImageDelete/VenueImageDelete.module.scss

```diff
@@ -0,0 +1,34 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: rem(32px);
+}
+
+.icon {
+  height: rem(80px);
+  width: rem(80px);
+}
+
+.header {
+  @include title3;
+}
+
+.spacer {
+  height: rem(32px);
+}
```

Comment:
It's not really that important, but it's better to have the classNames in a logic order, such as how it's displayed :)